_posts/2022-01-20-opensearch-as-a-service.markdown

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title:  "Partner Highlight: How to Offer OpenSearch as a Service using Virtuozzo DevOps PaaS"
+authors: 
+  - vlobzakov
```

Comment:
This needs an author bio. Create a file in `/_authors/` using a [format similar to this yaml file](https://raw.githubusercontent.com/opensearch-project/project-website/main/_authors/andhopp.markdown) with your info.

Reply:
Done: https://github.com/opensearch-project/project-website/commit/4970fa26450a7f3527cf53cce754ff488130f8ee